pkg/lightning/restore/check_info.go

```diff
@@ -84,23 +83,25 @@ func (rc *Controller) ClusterIsOnline(ctx context.Context) error {
 		rc.checkTemplate.Collect(Warn, passed, message)
 	}()
 
-	result := &api.RegionsInfo{}
-	err := rc.tls.WithHost(rc.cfg.TiDB.PdAddr).GetJSON(ctx, pdWriteFlow, &result)
-	if err != nil {
-		return errors.Trace(err)
-	}
-
-	for _, region := range result.Regions {
-		if region.WrittenBytes > OnlineBytesLimitation || region.WrittenKeys > OnlineKeysLimitation {
-			passed = false
-			regionStr, err := json.Marshal(region)
-			if err != nil {
-				return errors.Trace(err)
-			}
-			message = fmt.Sprintf("The write flow on cluster are more than expection, %s", string(regionStr))
-			return nil
-		}
-	}
+	// disabled the following code since it blocked some QA tests, it surpose to investigate a
```

Comment:
How about skip this function where it is called

Reply:
skip the function by remove it, since multiple lightning instance work on the cluster, check cluster workload is meaningless